framework/CodeInterfaces/RELAP7/RELAP7Interface.py

```diff
@@ -52,10 +52,16 @@ def generateCommand(self,inputFiles,executable,clargs=None,fargs=None,preExec=No
     if not found:
       raise IOError('None of the input files has one of the following extensions: ' + ' '.join(self.getInputExtension()))
     outputfile = 'out~'+inputFiles[index].getBase()
+
     if clargs:
       precommand = executable + clargs['text']
     else:
       precommand = executable
+
+    if executable.strip().endswith("py"):
+      # for testing
+      precommand = "python "+ precommand
+
```

Comment:
Why do we need this for the RELAP7 interface?
Should we have it also for other MOOSE based applications?

Reply:
This is only for testing the dynamic event tree....planning to remove the interface in the future